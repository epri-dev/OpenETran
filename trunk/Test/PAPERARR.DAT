4 41 3.0 1 1 0.005e-6 4.0e-6

conductor 1 8.5345 -1.146 13.8684e-3 0.0
conductor 2 8.9917  0.000 13.8684e-3 0.0
conductor 3 8.5345  1.146 13.8684e-3 0.0
conductor 4 7.7724  0.000 13.8684e-3 0.0

labelphase 0 G
labelphase 1 A
labelphase 2 B
labelphase 3 C
labelphase 4 N

surge     31.0e3 2.0e-6 103.6239e-6 0.0
pairs 2 0
poles 21

pipegap 840.0e3 436.89
pairs 1 4
poles all

pipegap 744.0e3 436.89
pairs 2 4
poles all

pipegap 840.0e3 436.89
pairs 3 4
poles all

pipegap 753.0e3 436.89
pairs 1 2
poles all

pipegap 1399.0e3 436.89
pairs 1 3
poles all

pipegap 753.0e3 436.89
pairs 2 3
poles all

ground 50.0 250.0 400.0e3 1.5441e-6 7.7724
pairs 4 0
poles 1 41

arrester 28.2e3 28.2e3 0.38 1.3e-6 0.0
pairs 1 4 2 4 3 4
poles 1 41

meter
pairs 2 4 1 4 3 4 4 0
poles 21
