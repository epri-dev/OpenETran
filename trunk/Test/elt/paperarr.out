  N   span     dT   Tmax
  4   3.00 5e-009 4e-006
Z-phase
  4.26924e+002  1.59342e+002  1.21006e+002  1.48481e+002
  1.59342e+002  4.30055e+002  1.59342e+002  1.57258e+002
  1.21006e+002  1.59342e+002  4.26924e+002  1.48481e+002
  1.48481e+002  1.57258e+002  1.48481e+002  4.21312e+002
Z-modal
  2.55270e+002  -1.42109e-014  6.00458e-014  -3.55271e-014
  -4.97380e-014  8.73943e+002  5.85894e-015  -4.26326e-014
  1.20283e-014  3.31311e-014  3.05918e+002  -2.83404e-014
  -1.42109e-014  1.13687e-013  -2.26878e-014  2.70085e+002
Modal Transformation (Ti)
  4.83863e-001  4.85595e-001  -7.07107e-001  -1.73418e-001
  -7.00770e-001  5.26272e-001  -3.44614e-017  -4.81622e-001
  4.83863e-001  4.85595e-001  7.07107e-001  -1.73418e-001
  -2.01679e-001  5.01431e-001  4.99183e-016  8.41364e-001
Power-frequency source at pole 1.
   #        vp        vm         i
   1       0.0       0.0       0.0
   2       0.0       0.0       0.0
   3       0.0       0.0       0.0
   4       0.0       0.0       0.0
Power-frequency source at pole 41.
   #        vp        vm         i
   1       0.0       0.0       0.0
   2       0.0       0.0       0.0
   3       0.0       0.0       0.0
   4       0.0       0.0       0.0
Meter at pole 21, from 1 to 4, max V   = -6.482669e+004
Meter at pole 21, from 2 to 4, max V   = 5.919896e+005
Meter at pole 21, from 3 to 4, max V   = -6.482669e+004
Meter at pole 21, from 4 to 0, max V   = 6.910336e+005
Arrester at pole 1, from 1 to 4 discharged -5.125202e+002 Amperes.
	Time operated: 5.550000e-007	Time of peak: 2.550000e-006
	Charge: -1.180948e-003	Energy: 3.347256e+001
Arrester at pole 1, from 2 to 4 discharged 1.506302e+004 Amperes.
	Time operated: 3.400000e-007	Time of peak: 2.645000e-006
	Charge: 3.832173e-002	Energy: 1.270358e+003
Arrester at pole 1, from 3 to 4 discharged -5.125202e+002 Amperes.
	Time operated: 5.550000e-007	Time of peak: 2.550000e-006
	Charge: -1.180948e-003	Energy: 3.347256e+001
Arrester at pole 41, from 1 to 4 discharged -5.125202e+002 Amperes.
	Time operated: 5.550000e-007	Time of peak: 2.550000e-006
	Charge: -1.180948e-003	Energy: 3.347256e+001
Arrester at pole 41, from 2 to 4 discharged 1.506302e+004 Amperes.
	Time operated: 3.400000e-007	Time of peak: 2.645000e-006
	Charge: 3.832173e-002	Energy: 1.270358e+003
Arrester at pole 41, from 3 to 4 discharged -5.125202e+002 Amperes.
	Time operated: 5.550000e-007	Time of peak: 2.550000e-006
	Charge: -1.180948e-003	Energy: 3.347256e+001
nr_iter = 0, nr_max = 0
Execution time [sec]: 0.062
